### DISCUSSION TITLE
npx vue-demi-switch 3 default

### DISCUSSION THREAD

**Discussion by hmendes00** [vueuse/vue-demi#101] [General]
I am not sure if this is a duplicated, but is there a way of having `npx vue-demi-switch 3 vue3` as default somehow?
Do I need to add it to the steps for when someone does an npm install?

Should I create a postinstall script for this? Thank you.

(I don't even know if this is the right place for asking that, but I didn't know where to go)

**Comment by koooge** [comment]
@hmendes00 I think so from looking at https://github.com/vueuse/vue-demi/blob/master/scripts/postinstall.js#L3


```json
  "scripts": {
    "postinstall": "vue-demi-switch 3 vue3"
  },
```

or  
  
```json  
  "devDependencies": {
    "vue": "npm:vue@3"
  },
  ```
